text/game/loadingtips.stringtable

```diff
@@ -416,32 +416,32 @@
     </Entry>
     <Entry>
       <ID>88</ID>
-      <DefaultText>Verwende die Tab-Taste, um interaktive Gegenstände in der Umgebung zu markieren.</DefaultText>
+      <DefaultText>Verwende die Tabtaste, um interaktive Gegenstände in der Umgebung anzuzeigen.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>89</ID>
-      <DefaultText>Wenn du die Försterhütte bei deiner Festung baust, erhältst du Zugang zu zahlreichen Kopfgeldaufträgen. Kopfgeld-Quests sind herausfordernde Kampfbegegungen, die wertvolle Belohnungen einbringen kann, darunter auch einzigartige Gegenstände.</DefaultText>
+      <DefaultText>Wenn du die Försterhütte bei deiner Festung baust, erhältst du Zugang zu zahlreichen Kopfgeldaufträgen. Kopfgeldqests sind herausfordernde Kampfbegegungen, die wertvolle Belohnungen einbringen können, darunter auch einzigartige Gegenstände.</DefaultText>
       <FemaleText />
     </Entry>
     <Entry>
       <ID>90</ID>
-      <DefaultText>NSCs mit goldenen Namen wurden von Backern gestaltet. Interagiere mit ihnen um die Möglichkeit zu haben diese einmaligen Geschichten zu erleben.</DefaultText>
+      <DefaultText>NPCs mit goldener Namensplakette wurden von Backern gestaltet. Interagiere mit ihnen, um die Möglichkeit zu haben, die einzigartige Geschichten jedes einzelnen dieser NPCs zu ergründen.</DefaultText>
```

Comment:
Hm, weiss nicht mehr, was wir bei den alten SSI Titeln geschrieben haben... Aber denke auch NSC geht in Ordnung.